sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/LoginSteps.java

```diff
@@ -127,6 +127,13 @@ public LoginSteps(
           webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(LOGOUT_BUTTON, 50);
         });
 
+    When(
+        "I navigate to {string} environment",
+        (String env) -> {
+          locale = env;
+          webDriverHelpers.accessWebSite(runningConfiguration.getEnvironmentUrlForMarket(locale));
+          TimeUnit.SECONDS.sleep(5);
```

Comment:
you can replace sleep with a wait for the logo or any category from left side menu to be displayed :D